docs/python/jupyter-support-py.md

```diff
@@ -105,6 +105,9 @@ You can offload intensive computation in a Jupyter notebook to other computers b
 
 To connect to a remote Jupyter server:
 
+0.a. Open vs code, make sure you have not opened any ipynb yet before proceeding.
```

Comment:
Not sure the open vs code suggestion is a good one. It's better to just start the jupyter server before you even start VS code.

I think I'd reword this as:
```
Start jupyter notebook or jupyter labs before running VS Code. 
Copy the URL presented under the text 'Or copy and paste one of these URLs'. They should have a form similar to: 
http://localhost:8889/?token=d9539e71c86b55da970823b15cf1ddafa14e003e1f67a42d
```

Reply:
Or maybe even more explicit:
```
Start 'jupyter notebook' or 'jupyter lab' before opening any .ipynb files inside of VS code.
```

Reply:
This is a very good suggestion.